### PR TITLE
Doc impro

### PR DESCRIPTION
**Pull request by Benedikt1992**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -205,128 +207,20 @@ docker-compose stop d2
...
-- **Full Client**
-This client supports all actions within the chain dependent on the rights of the participant's key.
-It will contain a complete copy of the blockchain.
-It is used by admissions to receive and validate transactions and to receive, validate and create new blocks.
-Doctors use the client to generate new transactions and to send them to their neighbors.
-- **Look-Up Client** `currently not implemented`  
-Meant to offer a user interface to enable search operations like Which vaccinations has a specific patient?, What are my upcoming vaccinations?... .
-This client can be used by any person and doesn't demand a valid key.
-Used with a key, it enables push notifications about upcoming vaccinations.
+**TODO** Any more?
```
Comment (Benedikt1992) [inline README.md]: @toniSta @alpreu Do you aware of any other points?

**Comment by Benedikt1992** [comment]
@alpreu @toniSta Any other topics?

File: README.md (hunk 1 above)
Comment (alpreu) [reply README.md]: I'd add that the Lookup Client needs a connection to the fullclient to work and does not store the chain by itself. 
Besides that I need to take the time to read the whole document and think about any other points

Comment (Benedikt1992) [reply README.md]: Sounds good but thats not a rela limitations. Maybe it ifts better into the section client types

File: consensus.md [hunk 2]
```diff
@@ -27,22 +27,22 @@ An `accept` judgement can be changed into a `deny` judgements at a later point i
...
  3. Is the block creation time older than every alternative block with the same index?
  
 If all 3 questions can be answered with yes (again based on the current state), the admission will create an `accept` judgement.
 At the same time every alternative branch (every sub-tree with the same index as root) will be updated to a `deny` judgement for every block in the branch.
-If one of the questions is answered with `false` the block hat and will get a `deny` judgement.
+If one of the questions is answered with no the block will get a `deny` judgement.
 
 As you can see every admission has only one active branch at every point in time.
-Since blocks and judgements are broadcasted in the network every admission will base its decision on the same information over time.
+Since blocks and judgements are broadcast in the network every admission will base its decision on the same information over time.
```
Comment (Benedikt1992) [inline consensus.md]: broadcasted

Comment (alpreu) [reply consensus.md]: the verb `broadcast` does not conjugate to `broadcasted` but `broadcast`, 
see http://conjugator.reverso.net/conjugation-english-verb-broadcast.html

Comment (Benedikt1992) [reply consensus.md]: https://dict.leo.org/pdf/flectionTable.pdf?kvz=4dkrADn71HeCbYCs67UP8lumz53RnXH31_iKUb_9hScDG0HfGwQNAs0iazuQS3MykZxvZlpa1HHSa1Dc6nR74kcHyaoRbdcTANyc1_09AVFVbsZcmwVqRAvHXGAGaFZ&lp=ende&lang=de

Comment (alpreu) [reply consensus.md]: looks like `broadcasted` is also accepted now by the big dictionaries. As `cast` is an irregular verb I'd still vastly prefer to use `broadcast` which is also the more common way of spelling (source: https://english.stackexchange.com/questions/33207/broadcast-or-broadcasted).

**Pull request merged**